packages/chart/src/components/AreaChart.jsx

```diff
@@ -42,38 +36,19 @@ const AreaChart = props => {
     }
   }
 
-  const getXAxisDates = brushDataSet => {
-    if (undefined === brushDataSet || !brushDataSet) return
-    let XAxisBrushDates = []
-    brushDataSet.forEach(function convertDateTimeNumber(key, value, brushDataSet) {
-      let tmp = getXValueFromCoordinate(xScale(value))
-      let date = formatDate(tmp)
-      XAxisBrushDates.push(date)
-    })
-    return XAxisBrushDates
-  }
-
   const handleX = d => {
     return config.xAxis.type === 'date' ? xScale(parseDate(d[config.xAxis.dataKey], false)) : xScale(d[config.xAxis.dataKey])
   }
 
   const handleY = (d, index, s = undefined) => {
-    return isBrush ? yScale(d[s.dataKey]) / 4 : yScale(d[s.dataKey])
-  }
-
-  // prevents duplicate brush handles being rendered
-  const getFirstBrushHandleOnly = (children, index) => {
-    if (index === 0) {
-      return children
-    }
-    // else dont return the other brush handles
+    return yScale(d[s.dataKey])
   }
 
   return (
     data && (
       <svg>
         <ErrorBoundary component='AreaChart'>
-          <Group className='area-chart' key='area-wrapper' left={Number(config.yAxis.size)} top={isBrush ? yMax * 1.3 : 0}>
+          <Group className='area-chart' key='area-wrapper' left={Number(config.yAxis.size)} top={0}>
```

Comment:
I think we need to consider the spacing for the x axis label here. I'm getting overlapping when adjusting the x axis size with the x axis label and the brush slider. Similarily I'm seeing overlap with the ticks. We should try to account for some of the scenarios before merging.